src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,71 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+import duke.DukeException;
+import duke.util.Parser;
+
+/**
+ * Event task.
+ */
+public class Event extends Task {
+    private LocalDate at;
```

Comment:
Hello Wei Quan, I think it would be clearer to use a noun as a variable name, as the preposition 'at' could also be used in front of a location.
```suggestion
    private LocalDate startTime;
```

Reply:
Yup, I agree with the above comment.

Reply:
I agree with this and the similar comment made on the deadline class